lib/query.js

```diff
@@ -267,6 +267,34 @@ Query.prototype.matchFilter = function (field, value) {
   return self;
 };
 
+/**
+ * wrapper function for matchFilter, accepting either an object with `field` and `value` properties
+ * or an array containing such objects to be mapped on matchFilter
+ *
+ * @param {Object|Array} filters - Object or an array of objects with `field` and `value` properties
+ *
+ * @return {Query}
+ * 
+ * @throws {Error}
+ * @api public
+ *
+ * @example
+ * var query = client.createQuery();
+ * query.q({ '*' : '*' }).fq({field: 'id', value: 100})
+ * query.q({ '*' : '*' }).fq([{field: 'id', value: 100}, {field: 'name', value: 'John'}])
+ */
+ Query.prototype.fq = function (filters) {
+  const self = this
+  if (filters instanceof Array) {
```

Comment:
Array.isArray is the standard way to check this, I believe

Reply:
done 👍